components/server/src/prebuilds/prebuild-manager.ts

```diff
@@ -186,13 +186,13 @@ export class PrebuildManager {
             const { commitHistory, additionalRepositoryCommitHistories } =
                 await this.incrementalPrebuildsService.getCommitHistoryForContext(context, user);
 
-            const prebuildEveryNthCommit = project?.settings?.prebuildEveryNthCommit || 0;
-            if (!forcePrebuild && prebuildEveryNthCommit > 0) {
+            const prebuildInterval = project?.settings?.prebuilds?.prebuildInterval || 0;
```

Comment:
We should use the same deault as everywhere. Worth getting introducing a getter in gitpod-protocol

Reply:
resolved